api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -228,21 +229,32 @@ private void ensureDrugOrderAutoExpirationDateIsSet(Order order) {
 			((DrugOrder) order).setAutoExpireDateBasedOnDuration();
 		}
 	}
-
+	
 	private void ensureOrderTypeIsSet(Order order, OrderContext orderContext) {
```

Comment:
@tendomart can you do another round of review for this method?

Reply:
Yes @dkayiwa  .Let me do that .

Reply:
Yes @dkayiwa  .Let me do that .